### PR TITLE
Added an analyze script

### PR DESCRIPTION
**Pull request by jaydenseric**
Added a new analyze script to the package.json that makes it possible to inspect what modules make it into the published package and how efficiently it bundles in a webpack v2 project.

The script runs a compilation and builds the analyze test project (that only imports this library) with webpack and the webpack-bundle-analyzer plugin in production mode. Once complete the plugin opens  your browser to an interactive analysis at http://localhost:8888.

See [discussion](https://github.com/apollostack/apollo-client/issues/684#issuecomment-266983921).

### CODE REVIEW

**Comment by kamilkisiela** [comment]
It's cool but it could be done with [`source-map-explorer`](http://npmjs.com/package/source-map-explorer) without installing the `webpack` which makes this project larger in terms of size (we <3 the waiting after `$ npm install`).

**Comment by stubailo** [comment]
Looks like this is just entirely new code, so there should be no cost to merging it other than bigger install size?

I guess one way we could remove that issue would be by adding a nested `package.json` inside the `analyze` folder, but that doesn't _really_ seem worth it. @kamilkisiela how much of a problem would installing `webpack` during development cause for you?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Expect active development and potentially significant breaking changes in the `0
 
 ### vNEXT
 
-- ...
+- Added a new analyze script to the package.json that makes it possible to inspect what modules make it into the published package and how efficiently it bundles in a webpack v2 project. [PR #1044](https://github.com/apollostack/apollo-client/pull/1044)
```
Comment (helfer) [inline CHANGELOG.md]: We usually use present tense (i.e. "add" instead of "added") in our changelog entries.

Comment (stubailo) [inline CHANGELOG.md]: IMO this shouldn't go into the changelog at all, since it's not a user-visible change, right?

**Comment by helfer** [comment]
Thanks @jaydenseric, now we just have to use it! 🙂 
If someone has a better way of doing this, please feel free to make another PR!

**Pull request merged**